### PR TITLE
Improvements on contributions and transactions form

### PR DESCRIPTION
**Pull request by gagnieray**

### CODE REVIEW

File: ui/css/galette.css [hunk 1]
```diff
@@ -0,0 +1,737 @@
+/*
```
Comment (trasher) [inline ui/css/galette.css:1]: This whole file change seems huge for current PR, did I miss something?

Comment (trasher) [reply ui/css/galette.css:1]: I guess this is a rebase issue ;)

Comment (gagnieray) [reply ui/css/galette.css:1]: Indeed ! Old css came back from the dead :rofl:

Comment (trasher) [reply ui/css/galette.css]: The famous come back of the living dead CSS file! :rofl:

**Review by trasher — APPROVED** [review]

**Pull request merged**